.eslintrc.js

```diff
@@ -0,0 +1,43 @@
+module.exports = {
+  root: true,
+  env: {
+    browser: true,
+    jest: true,
+    es6: true
+  },
+  extends: [
+    'nava',
+    'plugin:react/recommended',
+    'plugin:jest/recommended',
+    'plugin:jest-dom/recommended',
+    'plugin:testing-library/recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:@typescript-eslint/recommended-requiring-type-checking',
+    "plugin:prettier/recommended"
+  ],
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true
+    },
+    ecmaVersion: 12,
+    project: ['./tsconfig.json'],
+    sourceType: 'module',
+    tsconfigRootDir: __dirname,
+  },
+  settings: {
+    react: {
+      version: "detect" // Tells eslint-plugin-react to autodetect the React version to use
+    }
+  },
+  plugins: [
+    'jest',
+    'jest-dom',
+    'react',
+    'testing-library',
+    '@typescript-eslint'
+  ],
+  rules: {
+    "react/react-in-jsx-scope": "off",
```

Comment:
are we setting React as a global variable and that's why we can disable this? i think last year we had to include React in all jsx files, so just curious

Reply:
Oops just read the part of your PR description where you explain this is recommended by Next.js. Hm... could we either only disable this for Next.js files, and not for our own pages/components, or could we set React as a global variable (no idea if that's a good idea)? I think it's pretty common to forget to include React in JSX/TSX 😅